### PR TITLE
[13.0][MIG ] github_connector_odoo: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
cc @tecnativa TT20654

Depends on:

- [x] #47

### CODE REVIEW

File: github_connector_odoo/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Github Connector - Odoo",
+    "summary": "Analyze Odoo modules information from Github repositories",
+    "version": "13.0.1.1.0",
```
Comment (sergio-teruel) [inline github_connector_odoo/__manifest__.py]: Put 13.0.1.0.0

File: github_connector_odoo/demo/github_organization.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (C) 2016-Today: Odoo Community Association (OCA)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
```
Comment (sergio-teruel) [inline github_connector_odoo/demo/github_organization.xml]: Should be noupdate="1"??

File: github_connector_odoo/models/github_organization.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+from odoo import fields, models
+
+
+class GithubOrganization(models.Model):
+    _inherit = 'github.organization'
+
+    runbot_parse_url = fields.Char(
+        string='URL For Runbot Ids', oldname='runbot_url')
```
Comment (sergio-teruel) [inline github_connector_odoo/models/github_organization.py]: Remove oldname

File: github_connector_odoo/models/github_repository.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+class GithubRepository(models.Model):
+    _inherit = "github.repository"
+
+    runbot_id_external = fields.Integer(
+        string="ID For Runbot",
+        readonly=True,
+        store=True,
+        compute="_compute_runbot_id_external",
+        oldname="ci_id_external",
```
Comment (sergio-teruel) [inline github_connector_odoo/models/github_repository.py]: remove

File: github_connector_odoo/models/odoo_license.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+    module_version_qty = fields.Integer(
+        string="Number of Module Versions",
+        compute="_compute_module_version_qty",
+        store=True,
+    )
+
+    website = fields.Char(string="Website")
+
+    image = fields.Binary(string="Icon Image")
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_license.py]: Use fields.Image

File: github_connector_odoo/models/odoo_module.py [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+    )
+
+    author_ids = fields.Many2many(
+        string="Authors",
+        comodel_name="odoo.author",
+        compute="_compute_author",
+        relation="github_module_author_rel",
+        column1="module_id",
+        column2="author_id",
+        multi="author",
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_module.py]: You can remove multi attribute

File: github_connector_odoo/models/odoo_module.py [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+        column2="module_version_id",
+    )
+
+    dependence_module_version_qty = fields.Integer(
+        string="Number of Module Versions that depend on this module",
+        compute="_compute_dependence_module_version_qty",
+        store=True,
+    )
+
+    image = fields.Binary(
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_module.py:92]: Use field.Image

File: github_connector_odoo/models/odoo_module.py [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+        string="Icon Image", compute="_compute_image", store=True, reaonly=True
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_module.py]: Typo error --> readonly

File: github_connector_odoo/models/odoo_module.py [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+    @api.depends("module_version_ids.image")
+    def _compute_image(self):
+        module_version_obj = self.env["odoo.module.version"]
+        for module in self:
+            version_ids = module.module_version_ids.ids
+            last_version = module_version_obj.search(
+                [("id", "in", version_ids)], order="organization_serie_id desc", limit=1
+            )
+            if last_version:
+                module.image = last_version.image
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_module.py]: add else:
module.image = False

File: github_connector_odoo/models/odoo_module_version.py [hunk 10]
```diff
@@ -0,0 +1,507 @@
...
+                    icon_path = full_current_icon_path
+                    resize = True
+                    break
+        if icon_path:
+            image_enc = False
+            try:
+                with open(icon_path, "rb") as f:
+                    image = f.read()
+                if resize:
+                    image_enc = tools.image_resize_image(
```
Comment (sergio-teruel) [inline github_connector_odoo/models/odoo_module_version.py]: I think that image_resize_image method does not exists...
see https://github.com/odoo/odoo/blob/559b3a464e42362002f3ed08d7dd25084fc2e175/odoo/tools/image.py#L265

File: github_connector_odoo/views/action.xml [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+<!--
+Copyright (C) 2016-Today: Odoo Community Association (OCA)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+    <!-- Source Code Part -->
+    <record id="action_odoo_module" model="ir.actions.act_window">
+        <field name="name">Odoo Modules</field>
+        <field name="type">ir.actions.act_window</field>
```
Comment (sergio-teruel) [inline github_connector_odoo/views/action.xml]: remove type

File: github_connector_odoo/views/view_odoo_license.xml [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+        </field>
+    </record>
+    <record id="view_odoo_license_form" model="ir.ui.view">
+        <field name="model">odoo.license</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
```
Comment (sergio-teruel) [inline github_connector_odoo/views/view_odoo_license.xml]: PLease read this... https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration
See widget name="web_ribbon"

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by Tardo** [comment]
Changes done!

**Comment by sergio-teruel** [comment]
@Tardo travis fails. please... do a rebase to get the new configuration....

**Comment by Tardo** [comment]
Rebased!

**Comment by pedrobaeza** [comment]
@Tardo please rebase this one

**Comment by Tardo** [comment]
rebased

**Comment by pedrobaeza** [comment]
@legalsylvain are you interested in reviewing this one?

**Review by legalsylvain — APPROVED** [review]
Code review. No test. LGTM. Thanks !

**Review by pedrobaeza — APPROVED** [review]
I have undo the change to `fields.Image`, as it only adds unneeded resizing, and added `attachment=True` for better space allocation + lighter DB

**Pull request merged**